internal/scan-manager/scanners/veracode/src/main/java/org/wso2/security/tools/scanmanager/scanners/veracode/config/VeracodeScannerConfiguration.java

```diff
@@ -0,0 +1,94 @@
+/*
+ *  Copyright (c) 2019, WSO2 Inc., WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package org.wso2.security.tools.scanmanager.scanners.veracode.config;
+
+import org.wso2.security.tools.scanmanager.scanners.common.config.Configuration;
+import org.wso2.security.tools.scanmanager.scanners.common.config.ScannerConfiguration;
+
+/**
+ * Veracode specific configuration class.
+ */
+public class VeracodeScannerConfiguration extends ScannerConfiguration {
+    private static volatile VeracodeScannerConfiguration instance;
+    private String outputFolderPath;
+    private String outputFilePath;
+    private String logFilePath;
+    private char[] apiKey;
+    private String apiId;
+
+    private VeracodeScannerConfiguration() {
+        apiKey = new char[0];
+    }
+
+    public static VeracodeScannerConfiguration getInstance() {
+        if (instance == null) {
```

Comment:
Shall we get rid of double checking, and do proper volatile instantiation. 
There is no point of lazy initializing this variable, since we anyway need configs to be initialized at the startup of the service.